python/examples/advanced_agents/indie_hacker_agent/letta/setup.sh

```diff
@@ -2,11 +2,11 @@
 
 # Create a virtual environment
 echo "Creating virtual environment..."
-python3 -m venv ~/.venvs/content_writing 
+python3 -m venv ~/.venvs/indie_hacker_agent 
```

Comment:
The setup script should check if the virtual environment already exists before creating it to avoid potential errors. Consider adding:
```bash
if [ -d "$HOME/.venvs/indie_hacker_agent" ]; then
    echo "Virtual environment already exists"
else
    python3 -m venv ~/.venvs/indie_hacker_agent
fi
```